test/packages/parallel/gcp/data_stream/compute/_dev/deploy/tf/env.yml

```diff
@@ -2,5 +2,7 @@ version: '2.3'
 services:
   terraform:
     environment:
+      # terraform dedicated variable
       - TF_VAR_GCP_PROJECT_ID=${GCP_PROJECT_ID}
+      - GCP_PROJECT_ID=${GCP_PROJECT_ID}
```

Comment:
`TF_VAR` is Terraform specific (as is the way Terraform accepts variables through environment variables); to prevent a dependency in `test-default-config` I think is better to have defined the pass through variable `GCP_PROJECT_ID` (which is a convention and how you would expect it to be called) and the `TF_VAR` format.

Even if it's true that the value is the same I would not consider this duplication as the variables have different usage.

Reply:
(I also wanted to quickly check if this was the root cause, so open to discussion)

Reply:
Question: is this ENV (`GCP_PROJECT_ID `) required by any part of Terraform code? If not, let's follow KISS.

Reply:
So it means that you don't need it in this file.